Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM cgr.dev/chainguard/nginx:1
```

Comment:
Just a question about the `:1` and if there is a concern about using `:latest` instead?